src/pages/LoginPage.tsx

```diff
@@ -35,40 +35,46 @@ const LoginPage = (): ReactElement => {
    * sign in is blocked, display the aapropriate error message
    */
   async function login() {
-    setErrorType(0); // No error message before they get a response back
+    setErrorType(LoginError.NONE); // No error message before they get a response back
     if (!isValidNuid(input)) {
       console.log("Invalid NUID when log in pressed.");
       setSmallErrMsg("Should be 9 digits.");
-      setErrorType(1);
+      setErrorType(LoginError.OTHER);
       return;
     }
-    let member = undefined;
-    try {
-      member = await fetchMember(input); // CHANGE TO API IMPL
-    } catch (e) {
-      setErrorType(4);
-      return;
-    }
-    if (!member) {
-      setErrorType(1);
-      setSmallErrMsg("Member does not exist.");
-    } else {
-      if (
-        whetherHasAccess(member) &&
-        member.lastName.toUpperCase() === lastName.toUpperCase()
-      ) {
-        localStorage.setItem("user", input);
-        setUserID(input);
-        navigate("/events");
-      } else if (!member.activeMember) {
-        setErrorType(2);
-      } else if (member.signInBlocked) {
-        setErrorType(3);
-      } else {
-        console.log("in 4?");
-        setErrorType(4);
-      }
-    }
+    await fetchMember(input)
```

Comment:
I like the change to use the enum on the pop-up, but while you change that could you make the change to the pop-up on EventCard.tsx -> it uses the pop-up to show an api error on creating an attendanceChange

Reply:
Oh just realized, the route for the login should match the last name and the nuid of the user, this fetchMember route I believe this matches on the uuid (unique hash added on each row of the member table), so it needs a different route that takes those values in, there's a useState with the input, but there's also a lastName useState as well that should be passed. Then the route should hit the route in the auth folder

Reply:
So there's currently no password or anything, just NUID?

Reply:
I'll rewire to use the auth, but that is kinda sus lol

Reply:
yeah its definitely not ideal, talking to Matt(our client) about it he did say he was fine with it, and there technically wan't any big personal information risks from his side to be worried about it (the personal info is just your name and all you can really do is submit attendance changes) , ideally this should be northeastern sso, to not even have to worry about it, but obviously that would take time to research/figure out, this is kinda like a get it out approach so last name and nuid was fine with him